File: SEP3_UserSystem/src/main/java/model/domain/Unit/User/UserInformation.java
package model.domain.Unit.User;

import java.util.Date;

public abstract class UserInformation extends User{
    private String UserName;
    private Date Birthday;

    public UserInformation(String id,String userName)
    {
        super(id);
        UserName = userName;
    }

    public String getUserName() {
        return UserName;
    }

    public void setUserName(String userName) {
        UserName = userName;
    }

    public Date getBirthday() {
        return Birthday;
    }

    public void setBirthday(Date birthday) {
        Birthday = birthday;
    }

    public void update(String userName,Date birthday) {
        UserName = userName;
        Birthday = birthday;
    }
}

File: SEP3_Client/Model/Unit/User/UserInformation.cs
using System;

namespace SEP3_Client.Model.Unit.User
{
    public abstract class UserInformation : User
    {
        public string UserName { get; set; }
        public DateTime Birthday { get; set; }
        
        public UserInformation(string id,string userName) : base(id)
        {
            UserName = userName;
        }
        
        public string GetUserName() 
        {
            return UserName;
        }

        public void SetUserName(string userName) 
        {
            UserName = userName;
        }

        public DateTime GetBirthday() 
        {
            return Birthday;
        }

        public void SetBirthday(DateTime birthday) 
        {
            Birthday = birthday;
        }

        public void Update(string userName,DateTime birthday) 
        {
            UserName = userName;
            Birthday = birthday;
        }
    }
}